.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 sudo: false
 cache: pip
+dist: trusty
```

Comment:
can you tell me again why we need to set this distro and we can't use a more recent one?

Reply:
it's because otherwise `travis` would show error like here: https://github.com/openwisp/django-loci/pull/21#issuecomment-521583444